src/actions/arcActions.ts

```diff
@@ -1063,10 +1063,15 @@ export function onReputationChangeEvent(avatarAddress: string, address: string)
   }
 }
 
-export function onProposalExecuted(avatarAddress: string, proposalId: string, executionState: ExecutionState, decision: VoteOptions, reputationWhenExecuted: number) {
+export function onProposalExecuted(avatarAddress: string, proposalId: string, decision: VoteOptions, reputationWhenExecuted: number) {
   return async (dispatch: Dispatch<any>, getState: () => IRootState) => {
     const proposal = getState().arc.proposals[proposalId];
 
+    const contributionRewardInstance = await Arc.ContributionRewardFactory.deployed();
```

Comment:
It is more robust to get the ContributionReward from the DAO (`Dao.getSchemes`), not the one deployed.

Reply:
https://github.com/daostack/alchemy/pull/122/commits/4744d57fded5c62ebdbe4de258a0eafb39baf33a